.github/workflows/kind.yml

```diff
@@ -186,6 +186,8 @@ jobs:
     - name: Debug
       if: ${{ failure() }}
       working-directory: framework
+      env:
+        HOSTED: ${{ matrix.hosted }}
```

Comment:
I'm not actually sure why we're not just putting this (and `deployOnHub`) at the `jobs` level?

https://github.com/stolostron/governance-policy-framework/blob/6531fc537ccab7b2f9b8bc6c082e5e6454c1c8d5/.github/workflows/kind.yml#L23-L25